server_side/.htaccess

```diff
@@ -6,4 +6,5 @@ ShibRequestSetting requireSession true
 # REQUIRE CSE login
 ShibRequestSetting entityID https://cas01.cs.washington.edu/idp/shibboleth
 
-Require shib-user ~ @cs.washington.edu$
+# Require user yourself@cs.washington.edu colleague@washington.edu
+Require user tschweiz@cs.washington.edu
```

Comment:
Does Mike not want access as well?

Reply:
I think I remember that we agreed that the PI would have access. You raise a good point however that it might be more practical to also give him and Rene access so they don't depend on me.